### DISCUSSION TITLE
OCI ansible task for artifact upload in ArtifactUpload/DataPipeline

### DISCUSSION THREAD

**Discussion by vj4a** [Sunbird-Ed/Community#650] [Installation] [answered]
Hello @santhosh-tg, @gandham-santhosh,

**Working on SunbirdED installation in OCI**

Need a quick confirmation, We are facing a blocker in running pipelines under ArtifactUpload/DataPipeline because of missing ansible task [artifact upload] based on cloud provider for OCI.
https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/release-5.2.0/ansible/artifacts-upload.yml

Whereas for pipelines under ArtifactUpload/Core, the playbook has task for OCI.
https://github.com/project-sunbird/sunbird-devops/blob/release-6.0.0/ansible/artifacts-upload.yml

Kindly confirm if we are looking in the right branch [release-5.2.0] for  ArtifactUpload/DataPipeline and the ansible task is indeed missing.
We'll add it and raise a PR accordingly.

**Comment by santhosh-tg** [comment, marked as answer]
Hi @vj4a  there is a PR already for the OCI related changes to the branch release-5.2.0

https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/pull/1631

@manjudr @sowmya-dixit @SadanandGowda please let us know when we plan to merge the above PR

If you blocked until then , you can leverage the forked repo from the above PR or use this repo https://github.com/dicdiksha/sunbird-data-pipeline/tree/test-oci-5.1.0-github-99-deploy-job

**Reply by vj4a** [reply]
Thanks @santhosh-tg , we'll leverage the forked repo for now and proceed with artifact upload.

**Comment by mohitga** [comment]
@vj4a  is this issue resolved?

**Reply by vj4a** [reply]
@mohitga , This is resolved. Marking as closed.
